ui/generate-env-js.js

```diff
@@ -0,0 +1,43 @@
+const { dirname } = require('path');
+const fs = require('fs');
+
+const ENV_DEFAULTS = {
```

Comment:
It would be neat if this or something like this was tied into `generators/k8s.js` so that we don't hardcode ui config options  in there.

Reply:
Not blocking landing on this, feel free to open a bug an assign to me to make this happen.

Reply:
Yeah, I think that's a good next step.  Note, though, that `cd ui; yarn start` requires this information as well.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1576249